_posts/_2021-08-31-building-opensearch-1.1-distributions.markdown

```diff
@@ -0,0 +1,44 @@
+---
+layout: post
+title:  "Building OpenSearch 1.1 Distributions"
+authors: 
+  - dblock
+date: 2021-08-31
+categories: 
+  - technical-post
+twittercard:
+  description: "This post details how we produce reliable and repeatable distributions of OpenSearch."
```

Comment:
'we' is a non-specific plural pronoun. It could me Amazon, it could me you and your buddy, etc. 

Can you reformulate to avoid using 'we'? E.g. "This post details how the OpenSearch project produces reliable and repeatable distributions."